### PR TITLE
Add command to show all current user's products

### PR DESCRIPTION
**Pull request by sanakdam**

### CODE REVIEW

**Comment by adhikasp** [comment]
Maybe you should `git squash` your 2 commit because it's only very minor change

**Comment by sanakdam** [comment]
@adhikasp 👍

File: index.js [hunk 1]
```diff
@@ -171,9 +171,31 @@ function start(token, username){
...
 
+  /**
+   * Me Products
+   * See all Products created by current user
+   */
+  vorpal
+  .command('me products', 'See all products created by current user')
+  .action(function(args, callback) {
+    me(token, function(response){
+      var makerOf = response.body.user.makerOf;
```
Comment (Kristories) [inline index.js:182]: @sanAkdam `makerOf` or `maker_of`?

Comment (sanakdam) [reply index.js:182]: makerOf

Comment (Kristories) [reply index.js:182]: @sanAkdam do you have a test case for `response.body.user.makerOf`?

**Comment by sanakdam** [comment]
maybe you must read this link http://nodeguide.com/style.html

**Comment by Kristories** [comment]
Hi @sanAkdam,

I know about variables and properties should use lower `camelCase` capitalization. But, we are not talking about it (it about result).

I think your code will work if you change `response.body.user.makerOf` into `response.body.user.maker_of`.

**Pull request merged**